### PR TITLE
Integrated Marshal for client send. Fixed some style.

### PR DESCRIPTION
**Pull request by thelimeburner**

### CODE REVIEW

File: client/client-kv.c [hunk 1]
```diff
@@ -15,16 +15,21 @@
  ****************************************/
 //Returns the number of bytes to wait for
 
-
-
-int read_response(int sockfd){
+struct operation* read_line(int sockfd)
```
Comment (amygara) [inline client/client-kv.c:18]: plz send help, change formatting again
struct
operation\* read_line(int sockfd)
{

File: client/client-kv.c [hunk 2]
```diff
@@ -49,8 +53,12 @@ int read_response(int sockfd){
...
-	return atoi(msg);
+	gwkv_demarshal_client(msg, marshal_msg, status);
+	if (*status == -1) {
+		return 0;
+	} else {
+		return *marshal_msg;
+	}
 }
 
 int main(int argc, char ** argv)
```
Comment (amygara) [inline client/client-kv.c:64]: int
main(int argc, char *\* argv)
{

**Comment by amygara** [comment]
As soon as @huyang1988 fixes a couple things, demarshal_client should be up. You can see the code now at PR #47

**Comment by phil-lopreiato** [comment]
Tests failed due to missing make target

```
cd client && make test
make[1]: Entering directory `/home/travis/build/gwAdvNet2015/gw-kv-store/client'
make[1]: *** No rule to make target `test'.  Stop.
make[1]: Leaving directory `/home/travis/build/gwAdvNet2015/gw-kv-store/client'
make: *** [test_client] Error 2
```

This is something you can fix in conjunction with https://github.com/gwAdvNet2015/gw-kv-store/issues/50#issuecomment-94621946

**Comment by amygara** [comment]
@sacul29 @kstats @jdk514 Has any more progress been made on this? @phil-lopreiato recently merged #55 which should have completely resolved your issues with marshaling.

**Comment by amygara** [comment]
Moved to #58

**Pull request closed without merging**